### ISSUE TITLE
Requirements and dependencies

### ISSUE THREAD

**Issue by IzzySoft** [IridiumIon/DroidManager#3]
This project sounds very interesting! But I miss the requirements, install instructions, and what OS is supported (from the `releases/` it looks like Windows-only?) Could you please add some "generic hints" to the `README.md` file? Thanks in advance!

**Comment by 0xFireball** [comment]
Hello!

The release is quite outdated. For now, the easiest way to use it is to download the source, load submodules, restore NuGet packages, and build. I will update the readme too.

For now, Windows only. I'm going to start working on cross-platform support soon, though!

**Comment by IzzySoft** [comment]
Thanks a lot! Looking forward to that. I'm running in an MS-free zone here, so Windows-software doesn't work for me. As said, DroidManager looks very promising – so I hope for a cross-platform version which can be installed easily by the end user :)

**Comment by 0xFireball** [comment]
I totally understand!
Right now it uses WPF, a Windows-only technology.
Avalonia, a free, cross platform implementation, is still in alpha, but I'll try to get a basic app out in the near future.

**Comment by IzzySoft** [comment]
Thanks for the update! I then keep waiting patiently until this ticket gets updated again. Maybe I'll send another heads-up to update it myself if it "sleeps" for a few month – but I certainly won't rush you :) Good luck, and fingers crossed!

**Comment by 0xFireball** [comment]
I'm starting work soon! I made a branch: https://github.com/IridiumIon/DroidManager/tree/avalonia-rewrite

If you're familiar with C#/XAML/Avalonia, feel free to help out!

**Comment by IzzySoft** [comment]
Unfortunately, I'm not familiar with any of those. C# again is a MS speciality (thoug available on *nix via Mono), and the other 2 I don't even have an idea of ;) Good luck then, and fingers crossed!

**Comment by 0xFireball** [comment]
@IzzySoft are you still be interested in seeing this? AvaloniaUI is now working on .NET Core support, so soon .NET Core will be a viable platform for cross platform UI!

**Comment by IzzySoft** [comment]
Sure! I hope that will work with Mono then, as .NET itself is Windows only. So please let me know once it's ready and instructions are available, so I can take a look!

Btw: The screenshot on the main README is broken.

**Comment by 0xFireball** [comment]
As I mentioned, I will be using [.NET Core](https://www.microsoft.com/net/download/core), which is a new cross-platform and open source .NET that I consider to be a successor of both Mono and .NET

It will certainly run on Linux!

**Comment by IzzySoft** [comment]
Looking forward to it then! Hope it doesn't have too many dependencies to install ;)

**Comment by 0xFireball** [comment]
I don't think so, you can install it pretty painlessly through your package manager!

See this page:
https://www.microsoft.com/net/core#linuxubuntu

**Comment by IzzySoft** [comment]
Thanks! Looks like about what I was afraid of: a 3rd party repo run by MS (see my note above on *I'm running in an MS-free zone here).* So I only learn what dependencies are drawn in when I've run the install… Well, guess when you're ready I'll give it a try in a VM to see what it will "draw in", and then report back (so you can include those details with your instructions).

Not sure yet whether I want that on my real machine then. But that doesn't mean I won't give your app a look (on that VM) and include it with [my listing](https://android.izzysoft.de/applists/category/named/network_admin_adb#group_947). So please, just let me know when it's ready :smiley_cat:
